external/fv3fit/fv3fit/_shared/stacking.py

```diff
@@ -11,40 +9,6 @@
 Z_DIM_NAMES = ["z", "pfull"]
 
 
-class StackedBatches(Sequence[xr.Dataset]):
```

Comment:
Note that this means we must now always use the `unstacked_dim: z` configuration for keras training, might want to give the team a heads up (though I think @spencerkclark probably changed his config to do this anyways for recent experiments?)

Reply:
Yes, or `[tile, x, y, z]` for convolutional training. This PR won't change that, it's currently the case in master.

Reply:
Yup my latest config takes this into account.